src/main.rs

```diff
@@ -105,22 +160,27 @@ unsafe fn u8(u8: u8) {
 }
 
 unsafe fn fishy() {
-    assert_eq!(String::from("><>"),
-    String::<>::from::<>("><>").chars::<>().rev::<>().collect::<String>());
+    assert_eq!(
+        String::from("><>"),
+        String::from("><>").chars().rev().collect::<String>()
+    );
 }
 
 unsafe fn union() {
-    union union<'union> { union: &'union union<'union>, }
+    union union<'union> {
+        union: &'union union<'union>,
+    }
 }
 
 unsafe fn special_characters() {
-    let val = !((|(..):(_,_),__@_|__)((&*"\\","??")/**/,{})=={&[..=..][..];})//
-        ;
+    let val = !((|(..): (_, _), __ @ _| __)((&*"\\", "??") /**/, {}) == {
+        &[..= ..][..];
+    });
     assert!(!val);
 }
 
 unsafe fn punch_card() -> impl std::fmt::Debug {
-        ..=..=.. ..    .. .. .. ..    .. .. .. ..    .. ..=.. ..
+    ..=..=.. ..    .. .. .. ..    .. .. .. ..    .. ..=.. ..
         ..=.. ..=..    .. .. .. ..    .. .. .. ..    ..=..=..=..
         ..=.. ..=..    ..=.. ..=..    .. ..=..=..    .. ..=.. ..
         ..=..=.. ..    ..=.. ..=..    ..=.. .. ..    .. ..=.. ..
```

Comment:
This introduces breaking changes

Reply:
Yes, please fix asap, I don't want enterprise users to stop using

Reply:
hmmm, what about just bumping the major version? keeping backwards compatibility here would add significant maintainance overhead :/

Reply:
Absolutely out of question, this is *the* major feature for hello-world.rs if it wouldn't be for this we'd lose all revenue and customers.

(fr tho, look at this: https://discord.com/channels/273534239310479360/274215136414400513/875935651177050162)